src/APIFunctions/2DPrinting.js

```diff
@@ -0,0 +1,80 @@
+import axios from 'axios'
+
+/**
+ * Return an array similar to python's range() function
+ * @param {Number} start
+ * @param {Number} end
+ */
+export const range = (start, end) => {
+  const length = end - start
+  return Array.from({ length }, (_, i) => start + i)
+}
+
+/**
+ * Returns an array of numbers from pages
+ * @param {string} pages
```

Comment:
document your variables e.g.
```suggestion
 * @param {string} pages The number of pages requested by the user.
```